week-1/refactor/find.js

```diff
@@ -1,13 +1,24 @@
 // Refactor the implementation of find to use a for...of loop
 
-function find(list, target) {
-  for (let index = 0; index < list.length; index++) {
-    const element = list[index];
-    if (element === target) {
+// function find(list, target) {
+//   for (let index = 0; index < list.length; index++) {
+//     const element = list[index];
+//     if (element === target) {
+//       return index;
+//     }
+//   }
+//   return -1;
+// }
+
+//my refactor implementation:This way, you iterate over the elements of the list using a for...of loop, making the code cleaner and more concise.
+function find(list, target){
+  let index = 0;
+  for(const element of list){
+    if (element === target){
       return index;
     }
+    index++;
   }
   return -1;
 }
```

Comment:
The code is all good and working, just a few issues of style.  These can usually be found if you go back to your code after you have written it and review it yourself.  Think about all the special cases and see if you can write the code to work without them because that often simplifies the code.